tests/unit/test_postgres_target_adapter.py

```diff
@@ -90,8 +90,16 @@ def test_build_snowshu_envars():
 def test_copy_replica_command():
     """Test whether copy replica command is correctly called"""
     pg_adapter = PostgresAdapter(replica_metadata={})
+    pg_adapter.stop_postgres = MagicMock()
+    # Skip copy if only one container
     pg_adapter.container = MagicMock()
-    pg_adapter.container.exec_run = MagicMock(return_value=(0, ANY))
+    pg_adapter.container.exec_run = MagicMock(return_value=MagicMock())
```

Comment:
I thought MagicMock default return_value was already another MagicMock. Would it be possible to eliminate it here and below?

Reply:
![image](https://github.com/Health-Union/snowshu/assets/117748172/d027e92c-6d56-4d97-835f-8d958c159e09)

Reply:
This is not tested here, there is an integration test that checks if the data transfer replica to replica works. On the data sizes we are using for that test no issues appeared even before